pkg/utils/helper.go

```diff
@@ -25,3 +27,10 @@ func FormatCreatedTime(timestamp string) (string, error) {
 		return fmt.Sprintf("%d day ago", days), nil
 	}
 }
+
+func TrimSpaceOfLoginInputs(loginView *login.LoginView) {
+	loginView.Server = strings.TrimSpace(loginView.Server)
+	loginView.Username = strings.TrimSpace(loginView.Username)
+	loginView.Password = strings.TrimSpace(loginView.Password)
+	loginView.Name = strings.TrimSpace(loginView.Name)
+}
```

Comment:
```suggestion

```


we don't need separate function to trim space of inputs. Since we can use `huh.Validate()` will work fine.